tests/integration/handler/handlerPayload.test.js

```diff
@@ -9,6 +9,7 @@ describe('handler payload tests', () => {
   beforeAll(() =>
     setup({
       servicePath: resolve(__dirname),
+      noPrependStageInUrl: false,
```

Comment:
Indeed the test location could be enhanced, I'll see into it once the review is green.

Reply:
Sounds good, I was unsure and this seemed the easiest spot.